appveyor.yml

```diff
@@ -23,6 +23,6 @@ artifacts:
   - path: out\windows-amd64\crc.exe
     name: crc windows executable
     type: File
-  - path: out\windows-amd64\crc-installer.zip
+  - path: out\windows-amd64\crc-windows-installer.zip
```

Comment:
I'd squash this commit in the one renaming the .zip file.

Reply:
Done!